common/src/cellular_pkthandler.c

```diff
@@ -423,7 +420,7 @@ static int _sortCompareFunc( const void * pElem1Ptr,
 
     compareValue = strncmp( pElement1Ptr->pStrValue,
                             pElement2Ptr->pStrValue,
-                            MAX( element1PtrLen, element2PtrLen ) );
+                            MIN( element1PtrLen, element2PtrLen ) );
```

Comment:
Thanks for the fix.

Reply:
That's why we need  to do unit test coding, not only for the test, but review the code flow.